src/engine/enginemaster.cpp

```diff
@@ -632,7 +634,8 @@ void EngineMaster::process(const int iBufferSize) {
             SampleUtil::applyRampingGain(m_pMaster, m_masterGainOld,
                                          master_gain, m_iBufferSize);
             m_masterGainOld = master_gain;
-            if (!m_bExternalRecordBroadcastInputConnected) {
+            if (m_pEngineSideChain &&
+                    !m_bExternalRecordBroadcastInputConnected) {
```

Comment:
Oh, I just realised that this if/copy is also done on line 566 and 603. You need to modify those too.